viewer/db.js

```diff
@@ -1703,9 +1714,17 @@ Db.loadFields = async () => {
   return Db.search('fields', 'field', { size: 10000 });
 };
 
-Db.getIndices = async (startTime, stopTime, bounding, rotateIndex) => {
+Db.defaultIndexPatterns = function (extraIndices) {
+  const results = [...new Set([...['sessions2-*', 'sessions3-*'], ...extraIndices])];
+  if (internals.debug > 2) {
+    console.log(`defaultIndexPatterns: ${results}`);
+  }
+  return results;
+};
+
+Db.getIndices = async (startTime, stopTime, bounding, rotateIndex, extraIndices) => {
```

Comment:
The `extraIndices` argument has been added to `Db.getIndices` so that the user-defined ones can be returned as well.